### PR TITLE
fix: correct signature of `eth_getWork`

### LINKED ISSUE #3348: eth_getWork is not spec compliant
I understand this method is yet to be implemented, but right now it takes an argument that is not part of the spec (I presume a copy/paste error).

Current docs: https://github.com/trufflesuite/ganache/blob/develop/src/chains/ethereum/ethereum/RPC-METHODS.md#eth_getwork
Spec: https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getwork

### PR DESCRIPTION
**Pull request by emilbayes**
fixes #3348

### CODE REVIEW

**Review by tenthirtyone — APPROVED** [review]

File: src/chains/ethereum/ethereum/src/api.ts [hunk 1]
```diff
@@ -1402,15 +1402,14 @@ export default class EthereumApi implements Api {
...
    *
-   * @param filterId - A filter id.
    * @returns The hash of the current block, the seedHash, and the boundary condition to be met ("target").
    * @example
    * ```javascript
-   * console.log(await provider.send("eth_getWork", ["0x0"] ));
+   * console.log(await provider.send("eth_getWork", [] ));
    * ```
    */
   @assertArgLength(1)
```
Comment (tenthirtyone) [inline src/chains/ethereum/ethereum/src/api.ts]: I was a bit early to this approval. Please update this line to:

`@assertArgLength(0)`

**Review by tenthirtyone — CHANGES_REQUESTED** [review]

**Review by jeffsmale90 — APPROVED** [review]
Other than @tenthirtyone 's comment, this looks 👍

**Comment by emilbayes** [comment]
The comment has been resolved @tenthirtyone

**Review by tenthirtyone — APPROVED** [review]

**Comment by tenthirtyone** [comment]
Ah, related: https://github.com/trufflesuite/ganache/issues/3283

**Review by davidmurdoch — APPROVED** [review]

**Pull request merged**